.circleci/config.yml

```diff
@@ -270,7 +270,6 @@ workflows:
       - validate-with-script:
           name: "check-github-workflows-concurrency"
           command: "./.ci/validation.sh check-github-workflows-concurrency"
-      - yamllint
```

Comment:
Not sure why you want to remove this job, definition of it is above

Reply:
Please move it to be first.
Right under 
```
cli-validation:
    jobs:
```
To be more clear that it is not mistake